css/question-page.css

```diff
@@ -0,0 +1,44 @@
+.quiz-page-container{
+    display: grid;
+    grid-template-columns: repeat(12, 1fr);
+    grid-auto-rows: minmax(100px, auto);
+} 
+
+.quiz-page{
+    grid-column: 1/13;
+    grid-row: 2/8;
+    color: var(--font-color);
+} 
+
+.quiz-page-container .header-comp{
+    background-color: var(--background-color);
+}
+
+.quiz-page-container .h2-tag{
+    margin-top: 3rem;
+    margin-bottom: 3rem;
+}
+
+.question-score{
+     width: 47rem;
+     margin-top: 3rem;
```

Comment:
This can be changed to variables later for consistent styling for margin and padding